AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -134,16 +144,22 @@ - (void)enqueueItem:(id<MSLog>)item {
                                            code:kMSACConnectionSuspendedErrorCode
                                        userInfo:@{NSLocalizedDescriptionKey : kMSACConnectionSuspendedErrorDesc}];
       [self notifyFailureBeforeSendingForItem:item withError:error];
-      [self completedEnqueuingLog:item withInternalId:internalLogId withSuccess:NO];
+      [self enumerateDelegatesForSelector:@selector(channel:didCompleteEnqueueingLog:withInternalId:)
```

Comment:
Suggestion for readability: write a helper method for each callback to write one line instead of four.
OR one helper method with `NSInvocation` usage to call selector with the list of arguments.